farasa/__base.py

```diff
@@ -19,7 +19,8 @@ class FarasaBase:
     __bin_lib_dir = Path(f'{__bin_dir}/lib')
 
     # shlex not compatible with Windows replace it with list()
-    __BASE_CMD = ['java', '-jar']
+    # set java encoding with option `-Dfile.encoding=UTF-8`
+    __BASE_CMD = ['java', '-Dfile.encoding=UTF-8', '-jar']
```

Comment:
I was afraid of manually managing such lists. Let us check if there is any other suggestions for Windows regarding `shlex`. Otherwise, we have no other options but following this approach.

Reply:
I think this is OK, for now no other options or user defined options pass in. fucntion `shlex.split()` also return a list